src/es/components/atoms/flatpickr/Flatpickr.js

```diff
@@ -96,18 +96,22 @@ export default class Flatpickr extends Shadow() {
         cursor: var(--cursor, pointer);
       }
       :host > div {
-        border: 1px solid black;
+        border: var(--border, 1px solid black);
         border-radius: var(--border-radius, 2.5rem);
-        padding: 6px 24px;
+        padding: var(--padding, 6px 24px);
+        font: var(--font);
+        color: var(--color, black);
       }
       :host .label {
-        align-items: center;
-        display: flex;
-        gap: 1em;
+        display: var(--display, block);
```

Comment:
doesn't this have sideeffects by default block? Also the props like justify, aslign, flex, gap would not work on block.